pkg/frontend/transport/roundtripper.go

```diff
@@ -39,7 +39,7 @@ func (b *buffer) Bytes() []byte {
 }
 
 func (a *grpcRoundTripperAdapter) RoundTrip(r *http.Request) (*http.Response, error) {
-	req, err := server.HTTPRequest(r)
+	req, err := server.WrapHTTPRequest(r)
```

Comment:
I still think this function name is misleading and should be fixed. (Eg. using ToHTTPRequest, FromHTTPRequest, or something better.)

Reply:
I am fine with doing that if we move it to the `httpgrpc` package, I only balked at the To/From naming when considering this because `server.ToHTTPRequest` was not descriptive enough and `server.HTTPGRPCToHTTPRequest` was unwieldy.

I think `httpgrpc.To/FromHTTPRequest` is a sweet spot.